src/collision/mod.rs

```diff
@@ -289,8 +291,15 @@ pub struct Contacts {
     /// The total tangent impulse applied to the first body in a collision.
     ///
     /// To get the corresponding force, divide the impulse by `Time<Substeps>`.
+    #[cfg(feature = "2d")]
     #[doc(alias = "total_friction_impulse")]
     pub total_tangent_impulse: Scalar,
+    /// The total tangent impulse applied to the first body in a collision.
+    ///
+    /// To get the corresponding force, divide the impulse by `Time<Substeps>`.
```

Comment:
I assume you mean this?
```suggestion
    /// To get the corresponding force, divide the impulse by `Time<Substeps>::delta_seconds()`.
```